test/src/main/java/org/apache/accumulo/test/ZombieScanIT.java

```diff
@@ -310,15 +327,14 @@ public void testMetrics() throws Exception {
       });
 
       // Two of the stuck threads should respond to interrupts on the server side and exit.
-      Wait.waitFor(() -> countActiveScans(c, table) == 6);
+      Wait.waitFor(() -> countActiveScans(c, serverType, table) == 6);
 
       Wait.waitFor(() -> getZombieScansMetric() == 6);
 
-      assertEquals(6, countActiveScans(c, table));
+      assertEquals(6, countActiveScans(c, serverType, table));
```

Comment:
Do you want to shutdown the ScanServer at the end of this test if it's being used?

Reply:
updated to stop scan servers in 91ad6d7